eng/pipelines/README.md

```diff
@@ -29,3 +29,5 @@ Internal release pipelines (see [release process docs](/docs/release-process)):
   * 🚀 internal [`microsoft-go-infra-release-go-images`](https://dev.azure.com/dnceng/internal/_build?definitionId=1151)
 * (4) [`release-innerloop-pipeline.yml`](release-innerloop-pipeline.yml)
   * 🚀 internal [`microsoft-go-infra-release-innerloop`](https://dev.azure.com/dnceng/internal/_build?definitionId=1348)
+* (5) [`update-azure-linux-pipeline.yml`](update-azure-linux-pipeline.yml)
+  * 🚀 internal [`microsoft-go-update-azure-linux`](https://dev.azure.com/dnceng/internal/_build?definitionId=1405)
```

Comment:
The pipeline name is missing `-infra` in it.

Reply:
we didn't name it with that, if required we can ask the team to rename the pipeline?

Reply:
Looks like I was able to rename it. (So the Markdown just needs to be updated.)

Reply:
```suggestion
  * 🚀 internal [`microsoft-go-infra-update-azure-linux`](https://dev.azure.com/dnceng/internal/_build?definitionId=1405)
```

Reply:
fixed